src/DbPostgreSQLAdapter/FieldValue.cpp

```diff
@@ -0,0 +1,495 @@
+#include "stdafx.h"
+#include "FieldValue.h"
+
+namespace{
+	bool isDateTimeNull(const std::chrono::system_clock::time_point& dateTime)
+	{
+		return dateTime == std::chrono::system_clock::time_point();
+	}
+
+	std::chrono::system_clock::time_point stringISOToDateTime(const std::string& dateTime)
```

Comment:
I suggest you put this into a helper instead of repeating it everywhere

Reply:
I'll move it